src/components/policy/policy_regular/include/policy/policy_manager_impl.h

```diff
@@ -1046,6 +1052,11 @@ class PolicyManagerImpl : public PolicyManager {
    */
   void OnPTUIterationTimeout();
 
+  /**
+   * @brief Check that application for PTU more than zero
+   */
+  bool HasApplicationForPTU();
```

Comment:
i think this method can be const

Reply:
@mked-luxoft  done [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)